File: missing-number/daehyun99.py
# Time: O(N)
# Space: O(1)
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        base = 0
        for i in range(len(nums)+1):
            base += i
        for num in nums:
            base -= num
        return base        

File: missing-number/daehyun99.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int value = 0;
        for (int i = 0 ; i < nums.size()+1 ; ++i) {
            value += i;
        }
        for (int num : nums) {
            value -= num;
        }
        return value;
    }
};
